From bc797eb1ebeb2e94573f1b9ad7ada21fe948d9dc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Diego=20Elio=20Petten=C3=B2?= <flameeyes@gmail.com>
Date: Sun, 15 May 2011 19:07:41 +0200
Subject: [gpgme PATCH] build-sys: link libassuan and libgpg-error in
 gpgme-tool as well.

Without this change, the binary would result "underlinked", and GNU gold
would fail to link it.
---
 src/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 18c8b3b..0b240ee 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -240,7 +240,7 @@ libgpgme_qt_la_LIBADD = $(gpgme_res) @LIBASSUAN_LIBS@ @LTLIBOBJS@ \
 endif
 
 noinst_PROGRAMS = gpgme-tool
-gpgme_tool_LDADD = libgpgme.la
+gpgme_tool_LDADD = libgpgme.la @LIBASSUAN_LIBS@ @GPG_ERROR_LIBS@
 
 status-table.h : gpgme.h extra-stati.h
 	cat $(builddir)/gpgme.h $(srcdir)extra-stati.h \
-- 
1.7.5.rc3

